src/test/java/com/epam/tc/hw2/TestExercise1HomePage.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.tc.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+
+public class TestExercise1HomePage {
+
+    private WebDriver webDriver;
+    private SoftAssertions softAssertions;
+    private final List<String> expectedNavigatorBarItems = new ArrayList<>(
+        List.of("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"));
+    private final List<String> expectedBenefitTexts = new ArrayList<>(List.of("To include good practices\n"
+        + "and ideas from successful\n"
+        + "EPAM project", "To be flexible and\n"
+        + "customizable", "To be multiplatform", "Already have good base\n"
+        + "(about 20 internal and\n"
+        + "some external projects),\n"
+        + "wish to get more…"));
+    private final List<String> expectedSidebarElements = new ArrayList<>(
+        List.of("Home", "Contact form", "Service", "Metals & Colors", "Elements packs"));
+    private final int expectedIconsNumber = 4;
+    private final String expectedUserName = "ROMAN IOVLEV";
+    private final String expectedPageTitle = "Home Page";
```

Comment:
Java constants should be all uppercase (check java style guide)

Reply:
I know that, but CheckStyle does not allow it:

-Abbreviation in name 'EXPECTED_BENEFITS' must contain no more than '3' consecutive capital letters. (23:32) [AbbreviationAsWordInName]
-Member name 'EXPECTED_BENEFITS' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'. (23:32) [MemberName]

Should I ignore chechstyle?

Reply:
You can try to declare the constants as static to avoid checkstyle error: https://checkstyle.sourceforge.io/config_naming.html#ConstantName